src/dotnet-aspnet-codegenerator/dotnet-aspnet-codegenerator.csproj

```diff
@@ -66,20 +66,19 @@
-    <SignedPackageFile Include="tools/$(TargetFramework)/any/toolassets/net461/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild.dll" Certificate="$(AssemblySigningCertName)" />
-    <SignedPackageFile Include="tools/$(TargetFramework)/any/Microsoft.VisualStudio.Web.CodeGeneration.Contracts.dll" Certificate="$(AssemblySigningCertName)" />
-    <SignedPackageFile Include="tools/$(TargetFramework)/any/Microsoft.Extensions.FileProviders.Abstractions.dll" Certificate="$(AssemblySigningCertName)" />
-    <SignedPackageFile Include="tools/$(TargetFramework)/any/Microsoft.Extensions.FileProviders.Physical.dll" Certificate="$(AssemblySigningCertName)" />
-    <SignedPackageFile Include="tools/$(TargetFramework)/any/Microsoft.Extensions.FileSystemGlobbing.dll" Certificate="$(AssemblySigningCertName)" />
-    <SignedPackageFile Include="tools/$(TargetFramework)/any/Microsoft.Extensions.Primitives.dll" Certificate="$(AssemblySigningCertName)" />
+    <SignedPackageFile Include="..\VS.Web.CG.Msbuild\bin\$(Configuration)\netstandard2.0\Microsoft.VisualStudio.Web.CodeGeneration.Msbuild.dll" PackagePath="tools/$(TargetFramework)/any/toolassets/$(TargetFramework)/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild.dll" Certificate="$(AssemblySigningCertName)" />
+    <SignedPackageFile Include="..\VS.Web.CG.Msbuild\bin\$(Configuration)\net461\Microsoft.VisualStudio.Web.CodeGeneration.Msbuild.dll" PackagePath="tools/$(TargetFramework)/any/toolassets/net461/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild.dll" Certificate="$(AssemblySigningCertName)" />
 
     <!-- Third-party assemblies -->
-    <SignedPackageFile Include="tools/$(TargetFramework)/any/Newtonsoft.Json.dll" Certificate="3PartyDual" />
+    <SignedPackageFile Include="$(PublishDir)Newtonsoft.Json.dll" PackagePath="tools/$(TargetFramework)/any/Newtonsoft.Json.dll" Certificate="$(AssemblySigning3rdPartyCertName)" />
 
-    <!-- Binaries from partner teams that should be signed already -->
-    <ExcludePackageFileFromSigning Include="tools/$(TargetFramework)/any/NuGet.Frameworks.dll" Certificate="$(AssemblySigningCertName)" />
-    <ExcludePackageFileFromSigning Include="tools/$(TargetFramework)/any/System.Runtime.CompilerServices.Unsafe.dll" Certificate="$(AssemblySigningCertName)" />
+    <!-- Binaries from other projects that should be signed already -->
+    <ExcludePackageFileFromSigning Include="$(PublishDir)Microsoft.Extensions.FileProviders.Physical.dll" PackagePath="tools/$(TargetFramework)/any/Microsoft.Extensions.FileProviders.Physical.dll" />
+    <ExcludePackageFileFromSigning Include="$(PublishDir)Microsoft.Extensions.FileSystemGlobbing.dll" PackagePath="tools/$(TargetFramework)/any/Microsoft.Extensions.FileSystemGlobbing.dll" />
+    <ExcludePackageFileFromSigning Include="$(PublishDir)Microsoft.Extensions.Primitives.dll" PackagePath="tools/$(TargetFramework)/any/Microsoft.Extensions.Primitives.dll" />
```

Comment:
Is this a change from before where we used to sign all the other packages that got pulled in?

Reply:
Yes. So each repo will sign in batches now. Before, these files came from the FileSystem repo unsigned and ended up in this package. With MicroBuild signing, they will be signed before the Scaffolding repo builds. Each repo only needs to codesign the binaries it produces.

Reply:
Cool sounds much better